src/index.js

```diff
@@ -137,14 +155,88 @@ const getPointsLeftByDay = async (sprint, start) => {
       pointsLeftByDay[i] = 0;
     }
   }
+  log.info(JSON.stringify({ numDaysSinceSprintStart }));
+
+  if (!INCLUDE_WEEKENDS) {
+    // remove weekend entries
+    let index = 0;
+    for (
+      const cur = moment(start);
+      index < pointsLeftByDay.length;
+      cur.add(1, "days")
+    ) {
+      if (isWeekend(cur)) {
+        pointsLeftByDay.splice(index, 1);
+      } else {
+        index++;
+      }
+    }
+  }
+
   return pointsLeftByDay;
 };
 
-const generateChart = async (data, labels, filenamePrefix) => {
-  const pointsPerDay = data[0] / (labels[labels.length - 1] - 1);
-  const constantLine = labels.map(
-    (label) => data[0] - pointsPerDay * (label - 1)
+/** generate the guideline. A flat line is shown across weekends if {@link INCLUDE_WEEKENDS} is set to true */
+const getGuideline = (start, end, initialPoints, numberOfWeekdays) => {
+  const pointsPerDay = initialPoints / numberOfWeekdays;
+
+  log.info(
+    JSON.stringify({
+      initialPoints,
+      numberOfWeekdays,
+      pointsPerDay,
+    })
+  );
+
+  const guideline = [];
+  const cur = moment(start);
+  const afterEnd = moment(end).add(1, "days"); // to include the end day data point
+  let prevDayIsWeekday = false;
+  for (let index = 0; cur.isBefore(afterEnd); index++, cur.add(1, "days")) {
+    // if not including the weekends, just skip over the weekend days
+    if (!INCLUDE_WEEKENDS) {
+      while (isWeekend(cur)) {
+        cur.add(1, "days");
+      }
+    }
+
+    if (index === 0) {
+      guideline[index] = initialPoints;
+    } else {
+      guideline[index] =
+        guideline[index - 1] - (prevDayIsWeekday ? pointsPerDay : 0);
+    }
+
+    prevDayIsWeekday = !isWeekend(cur);
+  }
+
+  return guideline;
+};
+
+const getChartLabels = (numberOfDays) => {
+  // cool way to generate numbers from 1 to n
+  return [...Array(numberOfDays + 1).keys()].map((i) => i + 1);
+};
+
+const getChartDatasets = async (sprint, start, end) => {
+  const numDaysInSprint = moment(end).diff(start, "days");
+  const numberOfWeekdays = getNumberOfWeekdays(start, end);
```

Comment:
Nit. To be consistent with above, `numWeekdays`?

Reply:
Also what about we just combine the two variables

```
  const numDaysInSprint = INCLUDE_WEEKENDS ? moment(end).diff(start, "days") : getNumberOfWeekdays(start, end);
```

Reply:
I don't think can combine because `getIdealBurndown` requires `numWeekday` and can't use `numDaysInSprint` even if `INCLUDE_WEEKENDS` is true